sktime/forecasting/neuralforecast.py

```diff
@@ -192,6 +196,8 @@ def __init__(
         random_seed=1,
         num_workers_loader=0,
         drop_last_loader=False,
+        optimizer=None,
+        optimizer_kwargs: dict = None,
```

Comment:
New arguments should be added at end of function, otherwise it may be a breaking change for users who uses keyword arguments as positional.